client/src/components/StoreList.tsx

```diff
@@ -117,18 +123,73 @@ function StoreList() {
     }
   };
 
+  // Scan Store's QR code
+  const storeQR = async () => {
+    if (window.location == window.parent.location) {
```

Comment:
why is this needed?  Can add comment or extract to a function

Reply:
resolved in #22